tests/foreman/ui/test_openscap.py

```diff
@@ -113,37 +258,14 @@ def test_search_policies(self):
 
         @Feature: OpenScap - Search.
 
-        @Assert: Whether searching OpenScap policies is possible.
-
-        @status: Manual
+        @Steps:
 
-        """
-
-    @stubbed()
-    def test_audit_adhoc_hosts(self):
```

Comment:
Removed as Ad-hoc audit is a future feature as per the github link https://github.com/OpenSCAP/foreman_openscap [future features]